plugins/languages/treesitter/rainbow-delimiters.nix

```diff
@@ -0,0 +1,212 @@
+{
+  pkgs,
+  lib,
+  config,
+  ...
+}:
+with lib; let
+  helpers = import ../../helpers.nix {inherit lib;};
+in {
+  # TODO those warnings have been added XX/XX/2023
+  # -> Remove them in ~ 1 month (oct. 2023)
+  imports =
+    mapAttrsToList
+    (
+      old: new:
+        mkRenamedOptionModule
+        ["plugins" "treesitter-rainbow" old]
+        ["plugins" "rainbow-delimiters" new]
+    )
+    {
+      enable = "enable";
+      package = "package";
+      strategy = "strategy";
+      query = "query";
+      disable = "blacklist";
+      hlgroups = "highlight";
+    };
+
+  options.plugins.rainbow-delimiters =
+    helpers.extraOptionsOptions
+    // {
+      enable = mkEnableOption "rainbow-delimiters.nvim";
+
+      package =
+        helpers.mkPackageOption
+        "rainbow-delimiters.nvim"
+        pkgs.vimPlugins.rainbow-delimiters-nvim;
+
+      strategy =
+        helpers.defaultNullOpts.mkNullable
+        (with types; attrsOf str)
```

Comment:
We should have an easy way to specify the 3 builtin strategires (global, local, noop). I'd suggest having an attrsOf (either enum rawType) to handle that

Reply:
Good idea ! Done :)